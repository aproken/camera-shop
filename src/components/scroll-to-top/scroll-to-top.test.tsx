import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ScrollToTop from 'react-scroll-to-top';

describe('Компонент ScrollToTop', () => {
  it('Отрисовывает кнопку при определенной позиции на странице', () => {
    const { container } = render(<ScrollToTop />);
    expect(container).toMatchSnapshot();
  });
});
