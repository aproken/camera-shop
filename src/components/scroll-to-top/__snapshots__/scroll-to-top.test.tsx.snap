// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент ScrollToTop Отрисовывает кнопку при определенной позиции на странице 1`] = `<div />`;
